docs/docs/integration/embedding.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Embed Dashboards"
+description: Embed Dashboards in your own applications
+sidebar_label: "Embed Dashboards"
+sidebar_position: 10
+---
+
+## Embedding dashboards
+
+### Introduction
+You can embed Rill dashboards in your own applications using iframe. This allows you to use Rill dashboards as a component.
+
+To embed a dashboard, the customer's backend needs to use a Rill service token to request an authenticated iframe URL from `admin.rilldata.com`. 
+The iframe URL is then used to embed the dashboard in the customer's application. Here is a diagram of the flow:
+
+```mermaid
+sequenceDiagram
+  participant A as ui.ezcommerce.com
+  participant B as api.ezcommerce.com
+  participant C as admin.rilldata.com
+  participant D as ui.rilldata.com/-/embed
+  participant E as node-4.us-east1.runtime.rilldata.com
+  A ->> B: Get iframe URL
+  B ->> B: Resolve the user's email <br />using ezcommerce's own auth
+  B ->>+ C: Get iframe URL for:<br />project="ezcommerce"<br />user="john@example.com"<br/>(uses Rill service token)
+  Note right of C: 1. Lookup deployment<br/>2. Generate JWT<br />3. Build iframe URL
+  C ->>- B: iframe URL
+  B ->> A: iframe URL
+  A ->> D: Render iframe URL
+  loop 
+    D ->> E: Request data <br />(gets host and JWT from<br /> iframe URL query)
+    E ->> D: Data with policies applied for<br />user="john@example.com"
+  end
+```
+
+### Create a service token
+Use rill cli to create a service token for your organization. You can create a new service account along with token for your organization using the following command:
+
+```bash
+
+rill service create <service_name> [--org "My Organization"]
+```
+
+To issue service token for an existing service account, use the following command:
+
+```bash
+rill service token issue <service_name> [--org "My Organization"]
```

Comment:
Can we also mention about  
1. How to revoke the service account tokens ? 
2. How to delete a service account
3. Does issuing a new token invalidates any old tokens ?

Reply:
unhid service account cmds, added docs and a reference to them here.